server/planning/assignments/assignments.py

```diff
@@ -344,6 +344,14 @@ def send_assignment_notification(self, updates, original=None, force=False):
         ):
             return
 
+        user = get_user()
```

Comment:
this is current user again, which imo is not the one who should be receiving the notification

Reply:
my bad,I forgot to push this file